file-formats/chdo/examples/zaendbeleg.chdo.json

```diff
@@ -0,0 +1,43 @@
+{
+  "formatVersion": "",
```

Comment:
```suggestion
  "formatVersion": "1",
```

Reply:
I think the problem is that the example CHDO does not have the correct name: z_aff_example_chdo.

This will also be the case for the other object types I recently uploaded

Reply:
Well, the match for the JSON Schema and data is based on the format version, e.g. chdo-v1.json is matched for (chdo) JSON data with formatVersion 1